==5220== Invalid read of size 4
==5220==    at 0x404F13: imprimirTFecha(rep_fecha*) (fecha.cpp:95)
==5220==    by 0x4040AF: main_fechaInicioTExposicion(rep_exposicion*) (principal.cpp:1171)
==5220==    by 0x401EEC: main (principal.cpp:412)
==5220==  Address 0x8 is not stack'd, malloc'd or (recently) free'd
==5220== 
==5220== 
==5220== Process terminating with default action of signal 11 (SIGSEGV)
==5220==  Access not within mapped region at address 0x8
==5220==    at 0x404F13: imprimirTFecha(rep_fecha*) (fecha.cpp:95)
==5220==    by 0x4040AF: main_fechaInicioTExposicion(rep_exposicion*) (principal.cpp:1171)
==5220==    by 0x401EEC: main (principal.cpp:412)
==5220==  If you believe this happened as a result of a stack
==5220==  overflow in your program's main thread (unlikely but
==5220==  possible), you can try to increase the size of the
==5220==  main thread stack using the --main-stacksize= flag.
==5220==  The main thread stack size used in this run was 8388608.
==5220== 12 bytes in 1 blocks are definitely lost in loss record 1 of 5
==5220==    at 0x4842F95: operator new(unsigned long) (vg_replace_malloc.c:483)
==5220==    by 0x404BF1: crearTFecha(unsigned int, unsigned int, unsigned int) (fecha.cpp:10)
==5220==    by 0x40237C: leerFecha(rep_fecha*&) (principal.cpp:489)
==5220==    by 0x403EB7: main_crearExposicion(rep_exposicion*&) (principal.cpp:1122)
==5220==    by 0x401E1F: main (principal.cpp:402)
==5220== 
==5220== 12 bytes in 1 blocks are definitely lost in loss record 2 of 5
==5220==    at 0x4842F95: operator new(unsigned long) (vg_replace_malloc.c:483)
==5220==    by 0x404BF1: crearTFecha(unsigned int, unsigned int, unsigned int) (fecha.cpp:10)
==5220==    by 0x40237C: leerFecha(rep_fecha*&) (principal.cpp:489)
==5220==    by 0x403EC3: main_crearExposicion(rep_exposicion*&) (principal.cpp:1125)
==5220==    by 0x401E1F: main (principal.cpp:402)
==5220== 
